src/callout.tsx

```diff
@@ -52,13 +58,20 @@ const Callout = ({
 }: Props): JSX.Element => {
     const variant = useThemeVariant();
     const {texts, t} = useTheme();
+
+    const isTitleObject = typeof title === 'object';
+
+    const titleElementType = isTitleObject ? title?.as || titleAs : titleAs;
+    const titleAriaLabel = isTitleObject ? title?.['aria-label'] : undefined;
+    const titleText = isTitleObject ? title?.text : title;
```

Comment:
hmm, I think the isTitleObject condition is not needed. This should be fine
```suggestion
    const titleElementType = title?.as || titleAs;
    const titleAriaLabel = title?.['aria-label'] || undefined;
    const titleText = title?.text || title;
```

Reply:
TS won't allow this, because you can access `title?.as` if title can be an string